### ISSUE TITLE
implement unit tests with testthat

### ISSUE THREAD

**Issue by jburkhardt** [jburkhardt/RAdwords#96]

**Comment by oguzhanogreden** [comment]
Hi! I think this could be a good way for me to get to know the codebase better. I'd like to give this a go. What do you think?

**Comment by mbannert** [comment]
Sure... tests welcome. @jburkhardt how should we proceed here? 
Guess standard Fork and PR workflow seems suitable to me... @oguzhanogreden You familiar with that?

**Comment by oguzhanogreden** [comment]
Yes. I'll get going at it soon then, and post an early update here.

**Comment by jburkhardt** [comment]
hi @oguzhanogreden 

Yes, you are welcome to collaborate!

I opened a new branch called [tests](https://github.com/jburkhardt/RAdwords/tree/tests)
We can use this branch as a feature branch for the development of the tests. I suggest that we use the **testthat** framework. 

Moreover, I created a first [commit](https://github.com/jburkhardt/RAdwords/commit/1fccee585f72bc15a1b10732b4cb1a6a70ef3373) on this branch with tests for the ```changeNames```function.

You can pull and push from the tests branch to implement your tests.

@mbannert Is that a convenient way to go?

**Comment by mbannert** [comment]
I'd prefer @oguzhanogreden does a fork of RAdwords and then PRs from his local repository. His contribution will be honored in any case, but it's easier with rights management and CRAN releases that way.

**Comment by jburkhardt** [comment]
@mbannert ok, I see your point. Then I merge the tests branch into master.

**Comment by oguzhanogreden** [comment]
I took a look at this and what came out is fairly rudimentary, if not unnecessary. See [here](https://github.com/oguzhanogreden/RAdwords/tree/dev).

I found the code organized quite differently than how I'm used to, so it was hard for me make good guesses as to what you had in mind. Perhaps I'll be better aligned when I hear from you.

I'll soon be working with the package quite closely, so I also took a look with that eye while I was checking it out. I'm curious if you have a roadmap, if so how it looks, etc. Let me know!
